tiptapy/__init__.py

```diff
@@ -121,9 +121,9 @@ def inner_render(self, node) -> str:
         fallback_url = attrs['src']['fallback']
         image_type = url2mime(image_url)
         fallback_type = url2mime(fallback_url)
-        image_src = f'<img src="{fallback_url}"/>'
+        image_src = f'<img src="{fallback_url} loading="lazy""/>'
```

Comment:
It would be good to merge #54 first as `image_src` has been modified in that PR.